BugByte-backend/src/test/java/com/example/BugByte_backend/repositories/userProfileTest.java

```diff
@@ -65,6 +65,25 @@ SELECT COUNT(* )
                 WHERE id = ?;
             """;
 
+    private static final String SQL_GET_POSITIVE_INTERACTIONS = """
+                SELECT
+                    COUNT(*)
+                FROM
+                    posts p JOIN upvotes uv ON p.id = uv.post_id
+                WHERE
+                    op_name = ?
+            """;
+
+    private static final String SQL_GET_NEGATIVE_INTERACTIONS = """
+                SELECT
+                    COUNT(*)
+                FROM
+                    posts p JOIN downvotes dv ON p.id = dv.post_id
+                WHERE
+                    op_name = ?
+            """;
+
+
 
```

Comment:
Where are these queries used for?

Reply:
they relate to the old implementation
should be removed